pkg/server/log.go

```diff
@@ -36,16 +37,19 @@ func (f *logFormatter) Format(entry *log.Entry) ([]byte, error) {
 func (s *Server) initLogs() error {
 	filePath := s.config.LogFilePath
 
-	file, err := os.OpenFile(filePath, os.O_TRUNC|os.O_CREATE|os.O_WRONLY, 0644)
-	if err != nil {
-		return err
+	rotatedLogFile := &lumberjack.Logger{
+		Filename:   filePath,
```

Comment:
Should we also set `LocalTime: true`? (https://github.com/natefinch/lumberjack#:~:text=LocalTime%20bool%20%60json%3A%22localtime%22%20yaml%3A%22localtime%22%60)